extension/postgres_scanner/src/postgres_functions.cpp

```diff
@@ -0,0 +1,59 @@
+#include "postgres_functions.h"
+
+#include "duckdb_functions.h"
+#include "postgres_storage.h"
+
+using namespace kuzu::function;
+using namespace kuzu::main;
+using namespace kuzu::common;
+
+namespace kuzu {
+namespace postgres_scanner {
+
+struct PostgresClearCacheBindData : public duckdb_scanner::DuckDBClearCacheBindData {
+
+    PostgresClearCacheBindData(DatabaseManager* databaseManager,
+        std::vector<LogicalType> returnTypes, std::vector<std::string> returnColumnNames,
+        offset_t maxOffset)
+        : DuckDBClearCacheBindData{databaseManager, std::move(returnTypes),
+              std::move(returnColumnNames), maxOffset} {}
+
+    std::unique_ptr<TableFuncBindData> copy() const override {
+        return std::make_unique<PostgresClearCacheBindData>(databaseManager, columnTypes,
+            columnNames, maxOffset);
+    }
+};
+
+static offset_t DuckDBClearCacheTableFunc(TableFuncInput& input, TableFuncOutput& output) {
```

Comment:
Shouldn't we call this PostgresClearCacehableFunc()?

Also since there is only one table function in this file you might just call it tableFunc();

Reply:
There is a naming conflict there. The TableFunc already has a field called tableFunc, so we can't simply name it as tableFunc.